content/pages/uses/internet-of-things.md

```diff
@@ -41,6 +41,17 @@ This definition has some direct consequences that open up into different areas o
 
 As you will see, **XMPP** provides a lot of support into all these areas, making it well suited for use within the area of *Internet of Things*. To check current developments within XMPP and IoT, check the [Work in progress](/uses/iot/work-in-progress) page. There is also a page containing [testimonials](/uses/iot/testimonials).
 
+## Realized example projects
+
+Projects listed which are adopting XMPP for Internet of Things:
+
+| Users         | Game / Community                | Description                            | Since |
```

Comment:
Maybe "Game / Community" is less appropriate and should be replaced by "System / Application" or something similar.

Reply:
Fixed!